app/Filament/Resources/GameResource/RelationManagers/MemoryNotesRelationManager.php

```diff
@@ -0,0 +1,156 @@
+use App\Models\User;
+use Filament\Forms;
+use Filament\Forms\Form;
+use Filament\Resources\RelationManagers\RelationManager;
+use Filament\Support\Enums\FontFamily;
+use Filament\Tables;
+use Filament\Tables\Table;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Model;
+
+class MemoryNotesRelationManager extends RelationManager
+{
+    protected static string $relationship = 'memoryNotes';
+
+    protected static ?string $title = 'Code Notes';
+
+    public static function canViewForRecord(Model $ownerRecord, string $pageClass): bool
+    {
+        /** @var User $user */
+        $user = auth()->user();
+
+        return $user->can('manage', MemoryNote::class);
+    }
+
+    public function form(Form $form): Form
+    {
+        return $form
+            ->schema([]);
+    }
+
+    public function table(Table $table): Table
+    {
+        $gameId = $this->ownerRecord->id;
+
+        $associatedUsers = User::whereHas('authoredCodeNotes', function (Builder $query) use ($gameId) {
+            $query->where('game_id', $gameId);
+        })->get();
+
+        return $table
+            ->recordTitleAttribute('address')
+            ->searchPlaceholder('Search (Body)')
+            ->modifyQueryUsing(fn (Builder $query) => $query->where('body', '!=', ''))
+            ->columns([
+                Tables\Columns\TextColumn::make('address_hex')
+                    ->label('Address')
+                    ->fontFamily(FontFamily::Mono)
+                    ->color('info'), // Add some visual distinction between the address and the body.
+
+                Tables\Columns\TextColumn::make('body')
+                    ->label('Body')
+                    ->fontFamily(FontFamily::Mono)
+                    ->wrap()
+                    ->html()
+                    ->formatStateUsing(fn (string $state): string => nl2br(e($state)))
+                    ->searchable(),
+
+                Tables\Columns\TextColumn::make('user.display_name')
+                    ->label('Author'),
+
+                Tables\Columns\TextColumn::make('updated_at')
+                    ->label('Date Updated')
+                    ->dateTime()
+                    ->toggleable(isToggledHiddenByDefault: true),
+            ])
+            ->filters([
+                Tables\Filters\SelectFilter::make('user_id')
+                    ->label('Author')
+                    ->options($associatedUsers->pluck('display_name', 'ID'))
+                    ->searchable(),
+            ])
+            ->headerActions([
+
+            ])
+            ->actions([
+                Tables\Actions\ActionGroup::make([
+                    Tables\Actions\EditAction::make()
+                        ->form([
+                            Forms\Components\Textarea::make('body')
+                                ->label('Body')
+                                ->autosize()
+                                ->required(),
+                            ])
```

Comment:
I confirmed that whitespace is respected in RALibretro from notes saved using this control.